aibolit/metrics/cognitiveC/cognitive_c.py

```diff
@@ -27,47 +27,45 @@
 
 
 class CognitiveComplexity:
-    def _traverse_childs(self, ast: AST, node: int, nested_level: int, method_name: str) -> int:
+    def _traverse_childs(self, node: ASTNode, nested_level: int, method_name: str) -> int:
         complexity = 0
-        for each_child in ast.tree.succ[node]:
-            complexity += self._get_complexity(ast, each_child, nested_level, method_name)
+        for each_child in node.children:
+            complexity += self._get_complexity(each_child, nested_level, method_name)
         return complexity
 
-    def _check_if_statement(self, ast, expr, nested_level: int, method_name: str) -> int:
+    def _check_if_statement(self, node: ASTNode, nested_level: int, method_name: str) -> int:
         """function to work with IfStatement block"""
         complexity = 0
-        all_childs = list([i for i in ast.tree.succ[expr]])
-        complexity += self._get_complexity(ast, all_childs[0], 0, method_name)
+        all_childs = list(node.children)
```

Comment:
May it be a good chance to rename `all_childs` to a more correct `all_children`?